src/commands/api-mesh/create.js

```diff
@@ -112,7 +113,10 @@ class CreateCommand extends Command {
 		if (secretsFilePath) {
 			try {
 				await validateSecretsFile(secretsFilePath);
-				data.secrets = await interpolateSecrets(secretsFilePath, this);
+				const publicKey = await getPublicEncryptionKey(imsOrgCode);
+				const secretsData = await interpolateSecrets(secretsFilePath, this);
```

Comment:
```suggestion
				const secretsData = await interpolateSecrets(secretsFilePath, this);
				const publicKey = await getPublicEncryptionKey(imsOrgCode);
```

I woluld switch them because if `interpolateSecrets` failed due to any error, there wont be any use for the publicKey. Lets avoid unnecessary fetches where possible.